src/content/community/releases/_index.en.md

```diff
@@ -4,6 +4,23 @@ title = "Releases"
 weight = 40
 +++
 
+## v0.12.2
+
+This is a bugfix release:
+
+* Globalnet metrics port will now be opened by default when Globalnet is deployed using `subctl`.
```

Comment:
The current sentence talks about deployment and the first thought i get in my mind is its "subctl join ...". But we are talking about "subctl cloud prepare ...".

So, please clarify that metrics port is opened by default when cloud prepare is done via "subctl"